src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +91,10 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={
+              // eslint-disable-next-line max-len
+              !title.trim() || !imgUrl.trim() || !imdbUrl.trim() || !imdbId.trim()
+            }
```

Comment:
```suggestion
            disabled={isValidForm() }
```
And then please move that logic to a function definition. This way we the code will be much cleaner and there should be no need for disabling eslint